dotnet/src/Connectors/Connectors.AI.OpenAI/Tokenizers/Settings/GPT3Settings.cs

```diff
@@ -17,12 +17,9 @@ internal static class GPT3Settings
     internal static Dictionary<(string, string), int> BpeRanks => s_bpeRanks.Value;
 
     /// <summary>Lazy load the cached encoding table (encoder.json).</summary>
-    private static readonly Lazy<Dictionary<string, int>> s_encoder = new(() =>
-    {
-        return JsonSerializer.Deserialize<Dictionary<string, int>>(EmbeddedResource.ReadEncodingTable())
-               ?? throw new AIException(AIException.ErrorCodes.InvalidConfiguration,
-                   "Encoding table deserialization returned NULL");
-    });
+    private static readonly Lazy<Dictionary<string, int>> s_encoder = new(() => JsonSerializer.Deserialize<Dictionary<string, int>>(EmbeddedResource.ReadEncodingTable())
+                                                                                ?? throw new AIException(AIException.ErrorCodes.InvalidConfiguration,
+                                                                                    "Encoding table deserialization returned NULL"));
```

Comment:
The spacing seems to have gone wonky here. Far too wide.
Happy to get rid of the braces and return, but can we keep JsonSerialier... on a new line, and tab everything back?

Reply:
reworked the indentation with [this commit](https://github.com/microsoft/semantic-kernel/commit/f6b0f079bc335f90677e5d003ed77e2669ff27a7).